README.md

```diff
@@ -2,26 +2,29 @@
 
 # abapGit webfont generator package
 
+Icons in abapGit are displayed using a specially build font. As a default, abapGit uses icons from the free distribution of [Font Awesome](https://fontawesome.com/). When adding other icons mind the license matters please. Glyphs must be licensed for free usage (MIT, CC, SIL OFL).
+
+Note: The font is not limited to icons. For example, it include the abapGit logo (Git icon + abapGit text).
+
 ## Build
 
-You need nodejs to be installed (version 10 at the time of package creation)
+On every commit and pull request, a Github action will build the abapGit icon font and corresponding CSS files. The files are available as an artifact in the action and you can download them from there.
 
-```
```

Comment:
I would leave the manual procedure as a subsection of docs. This might be useful in case of issues or further devs

Reply:
+1